index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./landing.css">
+    <link rel="stylesheet" href="https://storm-components.netlify.app/components.css" />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="stylesheet" href="./quiz-css/grid-layout.css">
+    <title>StormIZ</title>
+</head>
+
+<body>
+    <div class="quiz-container">
+        <nav>
+            <a href="">
+                <i class=" theme fab fa-twitter"></i>
+            </a>
+            <a href="">
+                <i class=" theme fab fa-github"></i>
+            </a>
+            <i id="toggle-theme" class=" theme fas fa-moon"></i>
+        </nav>
+        <section>
+            <h1 class="title"><i class="fas fa-bolt"></i> StormIZ</h1>
+            <h2 class="subtitle">A Quiz App</h2>
+            <h2 class="category-title">Tv Shows Quiz :</h2>
+            <div class="quiz-item">
+                <div class="card-container product-card card-shadow">
+                    <div class="quiz-img">
+                        <img class="card-img" src="https://flxt.tmsimg.com/assets/p8630071_b_h8_bc.jpg" alt="..." />
+                    </div>
```

Comment:
add text related to image inside the alt

Reply:
thanks for that i will change it